oe_content.post_update.php

```diff
@@ -49,3 +49,10 @@ function oe_content_post_update_00001(): void {
     }
   }
 }
+
+/**
+ * Add Country corporate vocabulary.
+ */
+function oe_content_post_update_00002(): void {
+  \Drupal::service('oe_content.op_skos_setup')->setup();
```

Comment:
We cannot assume that the module users didn't add another graph already. Let's change this to only add the country vocabulary.